src/isDisplayable.js

```diff
@@ -0,0 +1,11 @@
+const isDisplayable = (key) => {
```

Comment:
@grimxreaper Would it be possible to add data to keypads rather than use this function?

Reply:
Hm, I'm not sure what you mean...which data? Do you mean replacing lines 2-5 in `isDisplayable`?

Reply:
I mean, could we store some data on keypads so that we no longer use this function at all? This function evaluates  whether a key is displayable but we know  when we look at keypads we know which ones are and could put something there that we check instead of calling a function. Hopefully I'm making sense? @grimxreaper

Reply:
Yes that makes sense, I'm thinking of including something like this in keypads but not sure how because keypads is an array of objects, so I can't insert this there, I'd have to make another function I think within that `keypads` component.

```
  if (key) {
    return true;
  }
  return false;
```

Reply:
But I don't think that's what you're getting at. I think you probably mean adding a property onto keypads for every key except '=, AC, CE'?

Reply:
@grimxreaper Yes!! Can we think of a better name for that `operation` property?

Reply:
@bryzaguy hmmm, how about, expression? or equation? We aren't including the '=' so I'm not sure if calling it any of these names is clear enough.

Reply:
@grimxreaper What is the purpose of the new property? What does it mean if a key has that property set to true?

Reply:
The purpose of it is to indicate that it should be displayed as part of the equation...

Reply:
How do you want your code to read? It will show up as `if (button.something) { ...` where "something" is a name that lets you know this button should be displayed as part of equation. Could be similar to what it's replacing.

Reply:
@grimxreaper

Reply:
Great question. `if (button.display)` `if (button.show)` does either read well?

Reply:
Those are much better. Try reading it "if button display" or "if button show" doesn't quite read right. A good rule of thumb for Boolean properties to be an adjective, sometimes prefixed with a verb like "is". Do you think `if (button.isShowable) { ...` or `if (button.isDisplayable) { ...` would be easier to understand?

Reply:
If they are Boolean properties, make them adjectives prefixed with an "is", okay great! Yes I think I'll stick with the second one.

Reply:
Perfect

Reply:
I should correct myself. If the Boolean represents a characteristic or quality of something, make it an adjective. If you're saying what it literally is, nouns are cool (i.e. "isBoat"). Using a prefix verb like "is", "was", "has" is good for both cases so if conditions read well.